docs/tutorials/security-with-pro.rst

```diff
@@ -0,0 +1,656 @@
+        Linux kernel did not properly handle certain unusual packets from a 
+        paravirtualized network frontend, leading to a buffer overflow. An 
+        attacker in a guest VM could use this to cause a denial of service 
+        (host system crash) or possibly execute arbitrary code.
+      * cve-2023-3611
+        It was discovered that the Quick Fair Queueing network scheduler 
+        implementation in the Linux kernel contained an out-of-bounds write 
+        vulnerability. A local attacker could use this to cause a denial of 
+        service (system crash) or possibly execute arbitrary code.
+      [...] 
+
+So, simply by attaching to a subscription, we have immediate, automatic
+security coverage for our kernel. We didn't need to apply the patches, or
+reboot the system for the fixes to be applied.
+
+Now we can check on the security status of our machine once more by running
+``pro security-status`` again:
+
+.. code-block:: text
+
+    555 packages installed:
+        550 packages from Ubuntu Main/Restricted repository
+        5 packages from Ubuntu Universe/Multiverse repository
+
+    [...]
+
+    This machine is attached to an Ubuntu Pro subscription.
+
+    Main/Restricted packages are receiving security updates from
+    Ubuntu Pro with 'esm-infra' enabled until 2028. You have received 115 security
+    updates.
+
+    Universe/Multiverse packages are receiving security updates from
+    Ubuntu Pro with 'esm-apps' enabled until 2028. You have received 1 security
+    update.
+
+We are now fully up-to-date with all of the software and security updates for
+the packages on our VM, and Livepatch has taken care of the kernel CVE fixes
+for us.
+
+But what about the case where we are not using a fresh VM? Let's now consider
+a scenario that might affect our live system.
+
+CVEs, USNs, and pro fix
+=======================
+
+There are two types of vulnerabilities that could affect a system:
+`Common Vulnerabilities and Exposures <cve_>`_ (CVEs) and
+`Ubuntu Security Notices <usn_>`_ (USNs).
+
+We've seen how, just by attaching our Pro subscription and upgrading our
+machine, the Pro Client services have taken care of applying the available
+fixes for the CVEs that affected our VM. In fact, as long as we upgrade
+our machine periodically, these fixes will always be applied after they are
+released. The default configuration of ``unattended-upgrades`` runs upgrades
+for you daily and includes Pro security updates.
+
+However, there is always a period of time between a CVE or USN being reported,
+and a fix being released. In this case, we might reasonably want some way to
+find out if a vulnerability is affecting our system.
+
+In an upcoming release of the Pro Client, we will have a command that will show
+us a list of all CVEs that affect our system, and their status. For now, we can
+use the ``pro fix`` command to manually inspect and resolve both CVEs and USNs.
+
+.. note::
```

Comment:
I don't think we support `pro fix` on an airgapped scenario, as we need to send requests to the security API and as far as I know, there is no straightforward way to mirror that

Reply:
Noted, thanks for the info :)